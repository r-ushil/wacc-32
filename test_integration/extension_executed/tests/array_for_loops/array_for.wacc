begin
    int[] array = [1,2,3];

    for x : array do
        x = 2
    done ;

    int y = array[0] + array[1] + array[2];

    # set all elems to 2, exit 6
    exit y
end