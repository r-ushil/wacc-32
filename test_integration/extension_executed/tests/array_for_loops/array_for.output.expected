Compiler exit code: 0
--------- stdout ---------
--------------------------
Exit code: 6
